src/main/java/omero/gateway/util/Mask.java

```diff
@@ -168,4 +181,60 @@ private static int getFirstNonZeroInt(int[][] array) {
         return result;
     }
     
+    /**
+     * Breaks a one-dimensional array into 'length' chunks,
+     * forming a two-dimensional array, e.g.
+     * 
+     * int[] x = 0 1 2 3 4 5 6 7 8 9 
+     * 
+     * using length = 4 will be transformed to
+     * 
+     * int[][] y = 
```

Comment:
don't know might be worth looking into formatting options so this kind of comment renders better